src/services/ModuleService.ts

```diff
@@ -0,0 +1,118 @@
+import { Module } from 'src/types/Module';
+
+import { Response2 } from './HttpService';
+
+export type ModulesData = {
+  modules: Module[] | undefined;
+};
+
+export type ModuleData = {
+  module: Module | undefined;
+};
+
+const mockModules: Module[] = [
```

Comment:
Looks good! I have some ideas on what the list/get APIs will return. These are very subject to change:

`GET /modules` will return a list of modules with the 
number (order), title, overview. This will give the details you need from the All Modules View Figma screen.

`GET /modules/{id}` will return the module and its contents/events/associated deliverables. Contents and events need more requirements to be finalized. But what you have looks great!

Reply:
Excellent, thanks for the insight! I think this should be good enough to get the UI done.